twitter_ads/resource.py

```diff
@@ -78,10 +78,6 @@ def to_params(self):
 
             if isinstance(value, datetime):
                 params[name] = format_time(value)
-            elif isinstance(value, list):
-                if not value:
-                    continue
-                params[name] = ','.join(map(str, value))
```

Comment:
*Before* removing this, `components` is (**incorrectly**) represented as comma-separated objects wrapped in double quotes, example:
```python
{'card_uri': 'card://1372017094963830793',
 'components': "{'type': 'MEDIA', 'media_key': '13_1191948012077092867'},{'type': 'DETAILS', 'destination': {'type': 'WEBSITE', 'url': 'http://twitter.com/'}, 'title': 'Twitter Web'}",
 'created_at': '2021-03-17T02:49:00Z',
 'name': 'example',
 'updated_at': '2021-03-17T02:49:00Z'}
```

*After* removing this, it's represented properly
```python
{'card_uri': 'card://1372017094963830793',
 'components': [{'media_key': '13_1191948012077092867', 'type': 'MEDIA'},
  {'destination': {'type': 'WEBSITE', 'url': 'http://twitter.com/'},
   'title': 'Twitter Web',
   'type': 'DETAILS'}],
 'created_at': '2021-03-17T02:49:00Z',
 'name': 'example',
 'updated_at': '2021-03-17T02:49:00Z'}
```

@twitterdev/ads-api-sdks let's talk through the implications of this change

Reply:
For peace of mind I would try running the example scripts that exercise batch and audiences, since Resource is used in many places I think the primary thing here is testing.

Reply:
For peace of mind I would run batch and audience related example scripts and make sure this doesn't unintentionally change behavior for a different resource type

Reply:
Ack! That's fair. Will do

Reply:
> For peace of mind I would try running the example scripts that exercise batch and audiences, since Resource is used in many places I think the primary thing here is testing.

I manually tested batch TC from [batch_request.py](https://github.com/twitterdev/twitter-python-ads-sdk/blob/master/examples/batch_request.py).

```python
targeting_criterion_1 = TargetingCriteria(account)
targeting_criterion_1.line_item_id = 'itttn'
targeting_criterion_1.targeting_type = 'LOCATION'
targeting_criterion_1.targeting_value = '00a8b25e420adc94'
targeting_criterion_1.operator_type = 'EQ'

targeting_criterion_2 = TargetingCriteria(account)
targeting_criterion_2.line_item_id = 'itttn'
targeting_criterion_2.targeting_type = 'PHRASE_KEYWORD'
targeting_criterion_2.targeting_value = 'tech'
targeting_criterion_2.operator_type = 'EQ'


targeting_criteria_list = [targeting_criterion_1, targeting_criterion_2]
TargetingCriteria.batch_save(account, targeting_criteria_list)
```

The targeting was successfully created.

I manually tested audiences using [tailored_audience.py](https://github.com/twitterdev/twitter-python-ads-sdk/blob/master/examples/tailored_audience.py).

```python
audience = TailoredAudience.create(account, 'test TA')
email_hash = hashlib.sha256("test-email@test.com").hexdigest()
user = [{
    "operation_type": "Update",
    "params": {
        "users": [{
            "email": [
                email_hash
            ]
        }]
    }
}]
success_count, total_count = audience.users(user)
```

This successfully added one user.

Reply:
LGTM